src/main/io/gps.c

```diff
@@ -93,6 +93,7 @@ GPS_svinfo_t GPS_svinfo[GPS_SV_MAXSATS_M8N];
 
 static serialPort_t *gpsPort;
 static float gpsDataIntervalSeconds;
+static uint16_t gpsStamp = ~0; // Initialize to an invalid state
```

Comment:
How do you guarantee that the biggest number is an invalid state? Eventually `gpsStamp` grows so big that it will be ~0

Reply:
I'm guessing that if it exceeded 65535 it would roll over, and would pass the non-equality test when there is new data, the same way it would pass if it just incremented by 1.

Reply:
in this case both GPS and  client counters should start at zero - return false until first gps fix arrives.
The original idea was to return true on first call, for that this counter will init to 0 and 'client' to value one smaller: ~0 (or better 1 here and 0 in client)